node/make.js

```diff
@@ -59,7 +52,152 @@ target.loc = function() {
         }
     }
 
-    // Create the en-US resjson file.
+    // create the en-US resjson file.
     var enContents = JSON.stringify(strings, null, 2);
     fs.writeFileSync(path.join(strPath, 'resources.resjson'), enContents)
-}
\ No newline at end of file
+}
+
+var rp = function (relPath) {
```

Comment:
I moved the `rp` and `run` helper functions down. I had to add a few more helper functions. It seemed like the target.___ assignments were more important to see at the top of the file (rather than leading the file with the helper functions). Also it seemed like all of the helper functions should be grouped together.